src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1704,4 +1704,44 @@ public String getTextFromComment(Text commentText) {
         }
         return SanitizationHelper.sanitizeForCsv(comment.toString());
     }
+
+    /**
+     * Checks if bundle contains response from Instructor.
+     *
+     * @return
```

Comment:
Empty `@return` in Javadoc doesn't help. So you can either remove it or add something to it. I recommend u to remove it as u will find yourself writing the same thing as the javadoc summary.